src/sync.rs

```diff
@@ -94,8 +99,37 @@ impl<A> SyncArbiter<A>
 where
     A: Actor<Context = SyncContext<A>>,
 {
-    /// Start new sync arbiter with specified number of worker threads.
-    /// Returns address of the started actor.
+    /// Start new `SyncArbiter` with specified number of worker threads.
+    /// Returns the address of the started actor pool.
```

Comment:
>actor pool

From user perspective it is still just single actor though
You cannot really operate the pool itself

Reply:
I didn't write this doc, this was me fixing formatting .... I also don't quite understand your comment here.

Reply:
Originally it was
>the started actor.

You changed it to
>started actor pool.

Do not do formatting while chaning stuff then

Reply:
I missed that I did the word change at the end there, that's my mistake. I'll fix this.

Reply:
Update: I actually ended up adding a bit extra here to clarify the "single address -> multiple actors" situation, because it could be confusing why there is single actor/address, but sync arbiter has many threads.